backend/app/services/summary_service.py

```
@@ -0,0 +1,519 @@
+            logger.info(
+                f"[SummaryService] Triggering KB summary after doc summary: {kb_id}"
+            )
+            await self.trigger_kb_summary(kb_id, user_id, force=False)
+
+    def _aggregate_document_summaries(self, kb_id: int) -> Optional[str]:
+        """Aggregate completed document summaries under the knowledge base."""
+        documents = (
+            self.db.query(KnowledgeDocument)
+            .filter(
+                KnowledgeDocument.kind_id == kb_id,
+                KnowledgeDocument.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `KnowledgeDocument.is_active:` for truth checks

Replace with `KnowledgeDocument.is_active`

(E712)